src/schemes/fluid/viscosity.jl

```diff
@@ -4,7 +4,7 @@ function dv_viscosity(particle_system, neighbor_system,
                       v_particle_system, v_neighbor_system,
                       particle, neighbor, pos_diff, distance,
                       sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-    viscosity = viscosity_model(neighbor_system, particle_system)
+    viscosity = viscosity_model(particle_system, neighbor_system)
```

Comment:
wait, why is this now the `particle_system`? This should be the neighbors system, since we use the viscosity of the interacting system. When I want to interact with a boundary system with slip-condition, it should be no viscosity.

Reply:
because this dispatches to the neighbor_system viscosity for a boundary system

Reply:
Is this correct? IMO the default should be the viscosity model of the neighbor system, no?

Reply:
you are right